tango_ros_common/tango_ros_native/include/tango_ros_native/tango_ros_node.h

```diff
@@ -81,15 +84,25 @@ const std::string USE_FLOOR_PLAN_PARAM_NAME = "use_floor_plan";
 const std::string ENABLE_DEPTH = "enable_depth";
 const std::string ENABLE_COLOR_CAMERA = "enable_color_camera";
 const std::string PUBLISH_POSE_ON_TF_PARAM_NAME = "publish_pose_on_tf";
-const std::string TANGO_3D_RECONSTRUCTION_RESOLUTION_PARAM_NAME = "reconstruction_resolution_3d";
+const std::string TANGO_3DR_RESOLUTION_PARAM_NAME = "reconstruction_resolution_3d";
+const std::string TANGO_3DR_USE_SPACE_CLEARING_PARAM_NAME = "reconstruction_use_space_clearing";
+const std::string TANGO_3DR_MIN_NUM_VERTICES_PARAM_NAME = "reconstruction_min_num_vertices";
+const std::string TANGO_3DR_UPDATE_METHOD_PARAM_NAME = "reconstruction_update_method";
+const std::string TANGO_3DR_MAX_VOXEL_WEIGHT_PARAM_NAME = "reconstruction_max_voxel_weight";
+const std::string TANGO_3DR_FLOORPLAN_MAX_ERROR_PARAM_NAME = "reconstruction_floorplan_max_error";
```

Comment:
this parameter is really about the floorplan_max_error and only has an effect on the floorplan while the other have an effect on all 3D reconstruction feature. cfr https://developers.google.com/tango/apis/c/reconstruction/reference/group/config-params